flink-runtime/src/test/java/org/apache/flink/runtime/jobmanager/JobManagerMetricTest.java

```diff
@@ -51,9 +51,11 @@ public void testJobManagerMetricAccess() throws Exception {
 		Deadline deadline = new FiniteDuration(2, TimeUnit.MINUTES).fromNow();
 		Configuration flinkConfiguration = new Configuration();
 
-		flinkConfiguration.setString(ConfigConstants.METRICS_REPORTER_CLASS, "org.apache.flink.metrics.reporter.JMXReporter");
+		flinkConfiguration.setString(ConfigConstants.METRICS_REPORTERS_LIST, "test");
+		flinkConfiguration.setString(ConfigConstants.METRICS_REPORTER_PREFIX + "test.class", "org.apache.flink.metrics.reporter.JMXReporter");
```

Comment:
shouldn't we uses the ConfigConstants.METRICS_REPORTER_CLASS here instead of `.class` ? I know it blows up the line, but it still is a defined constant. (the same applies to ExecutionGraphMetricsTest)